src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala

```diff
@@ -90,15 +90,13 @@ trait EntityPage extends HtmlPage {
                           mbr match {
                             case dtpl: DocTemplateEntity =>
                               dtpl.companion.fold(<span class="separator"></span>) { c: DocTemplateEntity =>
-                                <a class="object" href={relativeLinkTo(c)} title={c.comment.fold("")(com => Page.inlineToStr(com.short))}></a>
+                                <a class="object" href={relativeLinkTo(c)} title={c.comment.fold("")(com => Page.inlineToStrForTitleTag(com.short))}></a>
```

Comment:
Could we lift `_.fold("")(com => Page.inlineToStrForTitleTag(com.short))` to a method? It's repeated at least three times.

Reply:
Yes, I was contemplating this as well. Will make it part of this PR.